### PR TITLE
Issue 349

### LINKED ISSUE #365: Create automated tests for snaps-hcp
We need to ensure that snaps-hcp is behaving as expected.
To address, issue #353 must be addressed too.

### PR DESCRIPTION
**Pull request by akadam**
#### Questions:
- Have you connected this PR to the issue it resolves? Yes
- Does the documentation need an update? No
- Does this add new dependencies? No
- Have you added unit or functional tests for this PR?

### CODE REVIEW

**Comment by akadam** [comment]
Run Tofino lab trial build

**Comment by akadam** [comment]
Run Tofino Environment Build

**Comment by akadam** [comment]
Run Tofino lab trial build

**Review by spisarski — APPROVED** [review]

File: playbooks/scenarios/lab_trial/data_inspection_basic.yml [hunk 1]
```diff
@@ -21,12 +21,18 @@
...
     exe_systemd: "{% if ae_srvc_state == 'none' %}False{% else %}True{% endif %}"
   tasks:
-    - name: Restart tps-tofino-ae with state {{ ae_srvc_state }}
+    - name: Stop tps-tofino-ae with state {{ ae_srvc_state }} for analytics engine
       systemd:
         name: tps-tofino-ae
         state: "{{ ae_srvc_state }}"
       when: exe_systemd | bool
 
+    - name: Set up packet ingestion for all packets on analytics engine
```
Comment (spisarski) [inline playbooks/scenarios/lab_trial/data_inspection_basic.yml:30]: Move this block to the top of pkt-flood.yml

**Review by spisarski — CHANGES_REQUESTED** [review]
I think this will fix CI

File: playbooks/scenarios/lab_trial/pkt-flood.yml [hunk 2]
```diff
@@ -76,3 +42,6 @@
         dst_ip: "{% if scenario_send_ip_version|int == 4 %}{{ receiver.ip }}{% else %}{{ receiver.ipv6 }}{% endif %}"
         dst_port: "{{ send_port }}"
       ok_status: 201
+
```
Comment (spisarski) [inline playbooks/scenarios/lab_trial/pkt-flood.yml:45]: We need to stop espcap.py here and a service call would make this very easy.

**Review by spisarski — COMMENTED** [review]
Also should stop espcap.py

**Review by spisarski — APPROVED** [review]

**Pull request merged**